GateIo.Net/Clients/RebateApi/GateIoRestClientRebateApiExchangeData.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using CryptoExchange.Net.Objects;
+using GateIo.Net.Clients.RebateApi;
+using GateIo.Net.Interfaces.Clients.RebateApi;
+using GateIo.Net.Objects.Models;
+using Microsoft.Extensions.Logging;
+
+namespace GateIo.Net.Clients.SpotApi
+{
+    /// <inheritdoc />
+    internal class GateIoRestClientRebateApiExchangeData : IGateIoRestClientRebateApiExchangeData
+    {
+        private readonly GateIoRestClientRebateApi _baseClient;
+        private static readonly RequestDefinitionCache _definitions = new RequestDefinitionCache();
+
+        internal GateIoRestClientRebateApiExchangeData(ILogger logger, GateIoRestClientRebateApi baseClient)
+        {
+            _baseClient = baseClient;
+        }
+
+        #region Get Server Time
+
+        /// <inheritdoc />
+        public async Task<WebCallResult<DateTime>> GetServerTimeAsync(CancellationToken ct = default)
```

Comment:
I assume this is only here for the time sync implementation. You can instead reference the Spot API implementation `GetServerTimeAsync` implementation for it. Not needed to have this implementation here. See for example this Binance client which also references another GetServerTimeAsync:
https://github.com/JKorf/Binance.Net/blob/master/Binance.Net/Clients/GeneralApi/BinanceRestClientGeneralApi.cs